virttest/nbd.py

```diff
@@ -97,6 +99,9 @@ def export_image(qemu_nbd, filename, local_image, params):
     if params.get('nbd_allocation_exported') == 'yes':
         cmd_dict['allocation_depth'] = '-A'
 
+    if params.get('nbd_export_readonly') == 'yes':
```

Comment:
Add description about the new param 'nbd_export_readonly' (refer to nbd_allocation_exported(line 46) above)

Reply:
Split nbd.py code change out to a new commit, i.e. your PR will include two commits: one is for nbd and the other is for qemu_storage